dependabot-core.gemspec

```diff
@@ -0,0 +1,20 @@
+# This is a placeholder gem to prevent namesquatting on https://rubygems.org/gems/dependabot-core
+# Any updates must be manually published to RubyGems.
+# It's excluded from the normal release process because it's not expected to change.
+
+Gem::Specification.new do |spec|
+    spec.name        = "dependabot-core"
+    spec.version     = "0.95.1"
```

Comment:
The last published version before this was `0.94.13`, so this probably should be `1.0.0` since it's a breaking change to drop all source.

However, I wanted to test to see if this would actually work w/o `gem build`/`gem publish` complaining about lack of source files, so I used this version and well, it worked and it's live. 🤷‍♂️